.github/workflows/continuous-integration.yml

```
@@ -0,0 +1,225 @@
+name: "Continuous Integration"
+
+on:
+  pull_request:
+    branches:
+      - "*.x"
+      - "master"
+  push:
+    branches:
+      - "*.x"
+      - "master"
+
```

This will fail the build when an extension fails to install
```suggestion

env:
  fail-fast: true
```